.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+<!--
+Thanks for submitting a change!
+Before opening this pull request, please fill out the following checklist:
+
+NOTE: If you tick an item with **API review required** please tag the @openshift/api-reviewers group and wait until member of this group review this pull request before merging.
+-->
+
+<!-- WRITE DESCRIPTION HERE -->
+
+---
+
+#### API change checklist
+
+- [ ] This change introduces a new API or changes an existing API in a way that is complaint with the [upstream API conventions](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md)  _(**API review required**)_
+- [ ] This change is deprecating an existing API and [deprecation guide](https://docs.openshift.com/container-platform/4.9/rest_api/understanding-api-support-tiers.html) was followed _(**API review required**)_
```

Comment:
i'm not seeing where in that linked doc we talk about how we notify people of deprecated apis, or how a customer can find the list of deprecated apis.

I also think there are two aspects to this:

1) ensuring the user facing documentation that announces api deprecations is updated(whatever that is....release notes, list of deprecated apis in the product doc, some api endpoint that returns the list of deprecated apis, etc)
2) whatever mechanical process we use to mark the api as deprecated (e.g. tags in the go code itself) - we need to define this for our apis, possibly by directly copying the upstream one, and then point to it.